example/simpleDemo/embedderDownloader.go

```diff
@@ -270,22 +271,33 @@ func main() {
         fmt.Printf("Downloaded embedder for %s platform, matching version : %s\n", platform, hashResponse.Items[0].Sha)
     }
 
+    _, err = unzip(".build/temp.zip", dir + "/.build/")
+    if err != nil {
+        log.Fatal(err)
+    }
+
     switch platform{
-        case "darwin-x64":
-            _, err = unzip(".build/temp.zip", dir + "/.build/")
-            if err != nil {
-                log.Fatal(err)
-            }
-            
-            _, err = unzip(".build/FlutterEmbedder.framework.zip", dir)
-            if err != nil {
-                log.Fatal(err)
-            }
+    case "darwin-x64":       
+        _, err = unzip(".build/FlutterEmbedder.framework.zip", dir + "/FlutterEmbedder.framework")
+        if err != nil {
+            log.Fatal(err)
+        }
 
-            fmt.Println("Unzipped:\n")
+    case "linux-x64":
+        err := os.Rename("libflutter_engine.so", dir + "/libflutter_engine.so")
```

Comment:
Missing `.build//`

Reply:
This is wanted. When you unzip you want to have libflutter_engine.so into the same directory as your main.go. At least this is what is done when we follow old README.